cmd/di.go

```diff
@@ -337,6 +337,7 @@ func (di *Dependencies) bootstrapNodeComponents(nodeOptions node.Options) {
 	tequilapi_endpoints.AddRoutesForServiceSessions(router, di.ServiceSessionStorage)
 	tequilapi_endpoints.AddRoutesForPayout(router, di.IdentityManager, di.SignerFactory, di.MysteriumAPI)
 	tequilapi_endpoints.AddRoutesForAccessPolicies(router)
+	tequilapi_endpoints.AddRoutesForNat(router, di.NATTracker)
```

Comment:
Aren't abbreviations supposed to be upper case?
```suggestion
	tequilapi_endpoints.AddRoutesForNAT(router, di.NATTracker)
```